### PR TITLE
Added api test

### PR DESCRIPTION
**Pull request by TheoSBafrali**
#### Test Coverage Data

* Before
* After

### CODE REVIEW

**Comment by harmony-ek** [comment]
Jenkins speaking; can one of the admins verify this patch?

File: scripts/api_test.sh [hunk 1]
```diff
@@ -0,0 +1,619 @@
...
+	b)
+		NETWORK="betanet"
+		declare -A PORT=( [POST]="http://s0.b.hmny.io:9500/" [GET]="http://e0.b.hmny.io:5000/" )
+		BLOCK_0_HASH=$(curl --location --request POST "http://l0.b.hmny.io:9500" \
+			  --header "Content-Type: application/json" \
+			  --data "{\"jsonrpc\":\"2.0\",\"method\":\"hmy_getBlockByNumber\",\"params\":[\"0x1\", true],\"id\":1}" | jq -r '.result.hash')
+		echo "BLOCK0HASH:"
+		echo "$BLOCK_0_HASH"
+		
+		SIGNED_RAW_TRANSACTION=$(node ../dapp-examples/nodejs/testSign.js)
```
Comment (TheoSBafrali) [inline scripts/api_test.sh]: should call apiTestSign.js

**Comment by xChickens** [comment]
test this please

**Comment by xChickens** [comment]
test this please

**Comment by xChickens** [comment]
Jenkins, test this please.

**Comment by xChickens** [comment]
Jenkins, retest this please.

**Pull request merged**